architecture/core.rst

```diff
@@ -385,6 +385,56 @@ context is activated.
         "node_ttb":f1 -> "node_large_l1";
     }
 
+Page table cache
+================
+Page tables used to map TAs are managed with the page table cache. When the
+context of a TA is unmapped, all its page tables are released with a call
+to ``pgt_free()``. All page tables needed when mapping a TA are allocated
+using ``pgt_alloc()``.
+
+A fixed maximum number of translation tables are available in a pool.  One
+thread may be executing a TA which needs all or almost all tables. This can
+block TAs from being executed by other threads. To make sure that all TAs
+eventually will be permitted to execute ``pgt_alloc()`` temporarily frees
+eventual tables allocated before waiting for tables to become available.
+
+The page table cache behaves differently depending on configuration
+options.
+
+**Without paging,** ``CFG_WITH_PAGER=n``
```

Comment:
I would probably do like this (makes it linkable also and you can remove the empty line ... 405).
```rst
Without paging (``CFG_WITH_PAGER=n``)                                                                                                                                                                              
-------------------------------------
```

Reply:
OK, I'll fix. The reason I didn't do it like this from the beginning was that I though it was a bit hard to see that it really was a subsection to "Page table cache"